appinventor/components/src/com/google/appinventor/components/common/YaVersion.java

```diff
@@ -618,8 +618,8 @@ private YaVersion() {
   // - Added ShowFilterBar property
   // For LISTPICKER_COMPONENT_VERSION 8:
   // - Added title property
-
-  public static final int LISTPICKER_COMPONENT_VERSION = 8;
+  // For LISTPICKER_COMPONENT_VERSION 9 - Added ItemTextColor, ItemBackgroundColor
+  public static final int LISTPICKER_COMPONENT_VERSION = 9;
```

Comment:
@kkashi01 You also have to update YOUNG_ANDROID_VERSION. I suggest that you merge the current master branch before working more on this file...